### PR TITLE
Reinforced window construction fixes

### PR DESCRIPTION
**Pull request by actioninja**
## About The Pull Request

Fixes a couple of issues related to window construction introduced by the tougher windows PR.
Also gives plastitanium and shuttle windows matching values to reinforced and plasmawindows like I intended to on the last PR.

## Why It's Good For The Game

fix

## Changelog
:cl:
fix: newly constructed reinforced windows start in the correct state, meaning they can be disassembled
fix: crowbar can no longer be used out of order on construction
fix: screwed down but not levered in windows can be unscrewed again
tweak: shuttle and plastitanium windows now have the proper values to match reinforced windows.
/:cl:

### CODE REVIEW

File: code/game/objects/structures/window.dm [hunk 1]
```diff
@@ -669,13 +669,13 @@
 	icon = 'icons/obj/smooth_structures/shuttle_window.dmi'
 	icon_state = "shuttle_window"
 	dir = FULLTILE_WINDOW_DIR
-	max_integrity = 100
+	max_integrity = 150
```
Comment (PKPenguin321) [inline code/game/objects/structures/window.dm]: .

File: code/game/objects/structures/window.dm [hunk 2]
```diff
@@ -669,13 +669,13 @@
...
 	wtype = "shuttle"
 	fulltile = TRUE
 	flags_1 = PREVENT_CLICK_UNDER_1
 	reinf = TRUE
 	heat_resistance = 1600
-	armor = list("melee" = 50, "bullet" = 0, "laser" = 0, "energy" = 0, "bomb" = 50, "bio" = 100, "rad" = 100, "fire" = 80, "acid" = 100)
+	armor = list("melee" = 90, "bullet" = 0, "laser" = 0, "energy" = 0, "bomb" = 50, "bio" = 100, "rad" = 100, "fire" = 80, "acid" = 100)
```
Comment (PKPenguin321) [inline code/game/objects/structures/window.dm]: .

File: code/game/objects/structures/window.dm [hunk 3]
```diff
@@ -698,13 +698,13 @@
 	icon = 'icons/obj/smooth_structures/plastitanium_window.dmi'
 	icon_state = "plastitanium_window"
 	dir = FULLTILE_WINDOW_DIR
-	max_integrity = 100
+	max_integrity = 200
```
Comment (PKPenguin321) [inline code/game/objects/structures/window.dm]: .

File: code/game/objects/structures/window.dm [hunk 4]
```diff
@@ -698,13 +698,13 @@
...
 	wtype = "shuttle"
 	fulltile = TRUE
 	flags_1 = PREVENT_CLICK_UNDER_1
 	reinf = TRUE
 	heat_resistance = 1600
-	armor = list("melee" = 50, "bullet" = 0, "laser" = 0, "energy" = 0, "bomb" = 50, "bio" = 100, "rad" = 100, "fire" = 80, "acid" = 100)
+	armor = list("melee" = 95, "bullet" = 0, "laser" = 0, "energy" = 0, "bomb" = 50, "bio" = 100, "rad" = 100, "fire" = 80, "acid" = 100)
```
Comment (PKPenguin321) [inline code/game/objects/structures/window.dm]: .

**Review by PKPenguin321 — COMMENTED** [review]
Undocumented changes as far as I can tell

**Comment by actioninja** [comment]
I just forgot to mention those, meant to include those in the last PR.

**Comment by senjx** [comment]
big gameplay changes like this are test merged exclusively because of this tbh, not sure why the trend for prs like this was skipped

**Pull request merged**